gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/EhrExtractHandlingIT.java

```diff
@@ -51,7 +51,8 @@
         "entry[0].resource.id",
         "entry[0].resource.identifier[0].value",
         "entry[*].resource.subject.reference",
-        "entry[*].resource.patient.reference"
+        "entry[*].resource.patient.reference",
+        "entry[44].resource.performer.reference"
```

Comment:
should we add a check in Elliots function for this resource instead of naming it explicitly?